Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/BehaviorData.xml

```diff
@@ -4688,6 +4688,24 @@
             <Chance value="1"/>
         </DamageResponse>-->
     </CBehaviorBuff>
+    <CBehaviorBuff id="AP_MothershipTaldarimMovementNerfOutsidePower">
+        <Alignment value="Negative"/>

```

Comment:
Medic's Restoration removes that. Recolor the info icon instead. I've removed the alignment on behaviors those removal by Restoration cause bugs

Reply:
The medic can remove buffs with the Permanent flag? Feels like a bug on the Medic's end. Will test it out, though.

Reply:
Removed the negative flag, tested and medics could no longer restore it away. Also tried tinkering with more filters on restoration, but that didn't work so undid those changes and just went with avoiding the negative flag.